### ISSUE TITLE
Create a cron workflow that keeps prescriptions up to date

### ISSUE THREAD

**Issue by fridex** [thoth-station/thoth-application#1830]
**Is your feature request related to a problem? Please describe.**

Some prescriptions might change over time. To maintain the database of prescriptions publicly, it would be great to keep the database up to date. For this purpose, we should create a cron workflow that will make sure prescriptions are up to date.

**Describe the solution you'd like**

- [ ] identify prescriptions that can change over time
- [ ] design a cronworkflow that will keep prescriptions up to date

**Comment by harshad16** [comment]
will it be something similar to prescriptions-gh-release-notes job https://github.com/thoth-station/thoth-application/pull/1826?

**Comment by fridex** [comment]
Yes - some parts of the gh-release-notes job can be reused to keep release notes up to date. It can be one part of the workflow.
